news/_posts/releases/2020-10-22-prestashop-1-7-7-0-rc-release.md

```diff
@@ -0,0 +1,108 @@
+- [Fix an exception when we try to add a customized product](https://github.com/PrestaShop/PrestaShop/issues/20671)
+- [actionProductCancel hook not triggered on standard refund](https://github.com/PrestaShop/PrestaShop/issues/20712)
+- [Fix the impossibility to create an order when the address is considered as soft deleted](https://github.com/PrestaShop/PrestaShop/issues/20666)
+- ["Send an email to the customer with the link to process the payment." doesn't work](https://github.com/PrestaShop/PrestaShop/issues/20687)
+- [Missing gift wrapping and recycled paper options in create order page](https://github.com/PrestaShop/PrestaShop/issues/21300)
+- [In order detail page, when adding a discount amount with a different currency than the default one, an incorrect value is displayed and an incorrect amount is calculated](https://github.com/PrestaShop/PrestaShop/issues/21323)
+- [An exception is displayed when removing a discount with free gift from an order](https://github.com/PrestaShop/PrestaShop/issues/21438)
+- [Incorrect Product's total when adding a product with a discount gift or changing the Shipping](https://github.com/PrestaShop/PrestaShop/issues/21566)
+- [Incorrect total when updating a quantity's product](https://github.com/PrestaShop/PrestaShop/issues/21659)
+
+**Back Office:**
+- [Ranges are asked while they shouldn't during carrier edition](https://github.com/PrestaShop/PrestaShop/issues/20993)
+- [Following migration, $_GET identifier parameters are missing on "View {..} Page](https://github.com/PrestaShop/PrestaShop/issues/20934)
+- [Features page - an exception is displayed](https://github.com/PrestaShop/PrestaShop/issues/20777)
+- [Stats page - No module has been installed](https://github.com/PrestaShop/PrestaShop/issues/21186)
+- [An error is displayed when disabling or enabling maintenance](https://github.com/PrestaShop/PrestaShop/issues/21430)
+- [Hook displayBackOfficeTop not being called on Login BO Page](https://github.com/PrestaShop/PrestaShop/issues/21379)
+
+
+**Front Office:**
+- [Wrong cart rules discount when using alternative currency](https://github.com/PrestaShop/PrestaShop/issues/20594)
+- [Incorrect total with Free gift + Percent discount cart rule](https://github.com/PrestaShop/PrestaShop/issues/20690)
+- [When we have two currencies enabled - the Currency block and the language block are not working anymore](https://github.com/PrestaShop/PrestaShop/issues/20950)
+
+**For developers**
+- [Make PrestaShop compatible with composer 2.0](https://github.com/PrestaShop/PrestaShop/pull/21609)
+- [Update behat and gerkhin to allow debugging in PHPStorm](https://github.com/PrestaShop/PrestaShop/pull/21532)
+
+## Changelog
+
+151 pull requests have been merged since the beta 2 release and 40+ issues have been fixed. Read the [Changelog](https://github.com/PrestaShop/PrestaShop/releases) for details. 
+This brings us to more than 1300 merged pull requests for the 1.7.7.0 milestone, it is definitely our biggest yet.
```

Comment:
I didin't update the total number of PRs but it probably increase as well

Reply:
Yep, 1405 https://github.com/PrestaShop/PrestaShop/pulls?q=is%3Apr+is%3Aclosed+milestone%3A1.7.7.0